src/components/Book.js

```diff
@@ -4,6 +4,7 @@ import AddBook from './AddBook';
 const Book = () => (
   <div className="Books">
     <BookList />
+    <div className="Line" />
```

Comment:
- [ ] As I mentioned above ☝️ please stick to this Zaplin design if you check the divider the height is not matched with Zaplin Design so kindly fix this issue as well 🙂

**Your Design**
![Screenshot from 2023-08-24 00-54-50](https://github.com/As1imwe-Mark/book_store/assets/45226848/f52dc09e-c529-4391-9420-6738ed789994)

**Zaplin Design**
![Screenshot from 2023-08-24 00-55-25](https://github.com/As1imwe-Mark/book_store/assets/45226848/b1f7e5f5-a183-45de-835c-d7493ddb5426)